src/scss/components/footer/_footer.scss

```diff
@@ -0,0 +1,116 @@
+$topFooterBgColor: $tertiary;
+$topFooterTextColor: $white;
+$topFooterIconsBgColor: $white;
+$topFooterIconsTextColor: $primary;
+$topFooterFont: $franklin-medium-cd;
+
+.footer {
+
+  &__top {
+    background: $topFooterBgColor;
+    line-height: calculateRem(48px);
+    padding-top: $globalVerticalMargin;
+    padding-bottom: $globalVerticalMargin;
+  }
+
+  &__title {
+    color: $topFooterTextColor;
+    font-family: $topFooterFont;
+
+    @include font-size(16px);
+
+    @include desktop {
+      padding-right: 10px;
+    }
+  }
+
+  &__newsletter {
+    margin-bottom: 0;
+
+    @include desktop {
+      display: flex;
+      padding-right: 0;
+    }
+
+    .text-input {
+      background: $white;
+      border: 1px solid $basicInputColor;
+      border-radius: 3px;
+      display: flex;
+      font-size: 1rem;
+      line-height: 2rem;
+      margin-bottom: 0;
+      padding: 7px 10px;
```

Comment:
envelope is not aligned with input and arrow  align-items: center; will help for icon inside "input container" https://snag.gy/DqsYUR.jpg